core/services/log/broadcaster.go

```diff
@@ -133,14 +142,19 @@ func (b *broadcaster) Start() error {
 	})
 }
 
-func (b *broadcaster) LatestHead() *models.Head {
-	return b.latestHeadFromDb
+func (b *broadcaster) LatestHeadNumber() null.Int64 {
```

Comment:
This naming is now a bit confusing but looks like its only used in tests. Lets add a comment

Reply:
changed it now to BackfillBlockNumber